9-react.ts+TDD/src/components/Button.test.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
1. we use `.spec.tsx` instead of `.test`
2. we use `it('should ...` instead of `test('...`